working-with-multiple-clusters.md

```diff
@@ -0,0 +1,196 @@
+
+```
+$ kubectx docker-desktop
+Switched to context "docker-desktop".
+$ kubectx remote-cluster
+Switched to context "remote-cluster".
+$ kubectx -
+Switched to context "docker-desktop".
+```
+
+Every subsequence command you run will be making calls to the corresponding Kubernetes cluster.
+
+## Working with Multiple Namespaces
+
+At this point, you're able to connect and work with multiple Kubernetes clusters from your workstation. But remember that a context is also referring to a namespace in Kubernetes. That namespace will be the default one when you don't specify a namespace in a kubectl command. For instance, your `docker-desktop` context has the `default` namespace configured. Then, every time you run `kubectl get pods`, it will return the list of pods from the `default` namespace.
+
+However, there are going to be times where you also need to work with different namespaces. Perhaps you're in charge of two applications, and each application uses a namespace to deploy the Kubernetes objects it needs. Every time you run a kubectl command, you'd have to specify the name of the namespace you want to use, like this:
+
+```
+$ kubectl get pods -n service-a
+$ kubectl get svc -n service-b
+```
+
+You might don't see a problem with the above commands. However, when you're coding, you might need to change namespaces a little bit quicker and avoid any confusion if you forgot to include the namespace's proper name. If you don't want to specify a namespace in every kubectl command you run, you'd have to modify the `~/.kube/config` file. However, there's a project called [kubens](https://github.com/ahmetb/kubectx) (from the same project creator as kubectx) that you could use to switch between different namespaces quickly.  kubens will update the Kubernetes context for you automatically. The above commands now can be like this:
+
+```
+$ kubens
+default
+service-a
+service-b
+$ kuebns service-a
+Context "docker-desktop" modified.
+Active namespace is "service-a".
+$ kubectl get pods
+$ kubens service-b
+Context "docker-desktop" modified.
+Active namespace is "service-b".
+$ kubectl get svc
+```
+
+As you've noticed, kubectx and kubens help you work with different clusters and namespaces, which is a little less problematic.
+
+But what if there's even a better way for developers? Let's see.
+
+## Developing with Contexts in Okteto
+
+Okteto has support for configuring the context and namespace your applications might need. Hence, you have consistency at coding time because all developers are going to use the same configurations. Also, you'll have fewer commands to run. Even if the existing commands from kubectcl or the kubectx utility, you still need to switch manually. Conversely, with Otketo, you can include this configuration in the YAML manifest.
```

Comment:
```suggestion
Okteto has support for configuring the context and namespace your applications might need. Hence, you have consistency at coding time because all developers are going to use the same configurations. Also, you'll have fewer commands to run, so you can focus on writing your application, not  on managing clusters. Even if the existing commands from kubectcl or the kubectx utility, you still need to switch manually. Conversely, with Otketo, you can include this configuration in the YAML manifest and forget about it.
```